### PR TITLE
Cleanup metadata: remove/rename event history references

### PR DESCRIPTION
**Pull request by oleewere**
# What changes were proposed in this pull request?
renamed remaining references: eventHistory -> metadata

## How was this patch tested?
not needed

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-LogSearch-Github-PR-Builder/84/
Test PASSed.

File: ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/converter/MetadataRequestQueryConverter.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+@Named
+public class MetadataRequestQueryConverter extends AbstractConverterAware<MetadataRequest, SolrQuery> {
+
+  @Override
+  public SolrQuery convert(MetadataRequest metadataRequest) {
+    SolrQuery metadataQuery = new SolrQuery();
+    metadataQuery.setQuery("*:*");
+
+    int startIndex = StringUtils.isNotEmpty(metadataRequest.getStartIndex()) && StringUtils.isNumeric(metadataRequest.getStartIndex())
+      ? Integer.parseInt(metadataRequest.getStartIndex()) : 0;
```
Comment (kasakrisz) [inline ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/converter/MetadataRequestQueryConverter.java]: I recommend let that NumberFormatException thrown so the user will know that a number must be specified.
That same for `metadataRequest.getPageSize()`

File: ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/manager/MetadataManager.java [hunk 2]
```diff
@@ -119,36 +119,36 @@ public void deleteMetadata(String id) {
...
     Collection<LogsearchMetaData> configList = new ArrayList<>();
 
     for (SolrDocument solrDoc : solrList) {
-      LogsearchMetaData eventHistoryData = new LogsearchMetaData();
-      eventHistoryData.setFiltername("" + solrDoc.get(FILTER_NAME));
-      eventHistoryData.setId("" + solrDoc.get(ID));
-      eventHistoryData.setValues("" + solrDoc.get(VALUES));
-      eventHistoryData.setRowType("" + solrDoc.get(ROW_TYPE));
+      LogsearchMetaData metadata = new LogsearchMetaData();
+      metadata.setFiltername("" + solrDoc.get(FILTER_NAME));
```
Comment (kasakrisz) [inline ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/manager/MetadataManager.java]: 1. `String.valueOf(solrDoc.get(FILTER_NAME))` can be used here.
2. If `solrDoc.get(FILTER_NAME)` returns null the result string will be `"null"`. Is it intended?

File: ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/converter/MetadataRequestQueryConverter.java (hunk 1 above)
Comment (oleewere) [reply ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/converter/MetadataRequestQueryConverter.java]: @kasakrisz it was only a rename on request query converter, but as some references are changed here, github show this as a new file. that can be fixed in an another PR, i think that is used in other places as well

File: ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/manager/MetadataManager.java (hunk 2 above)
Comment (oleewere) [reply ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/manager/MetadataManager.java]: same as above. also note that part will be the next to rewrite as Tobi will start to use metadata collections for storing user settings.

Comment (oleewere) [reply ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/manager/MetadataManager.java]: also filterName is required field in Solr so we won't get null here

**Pull request merged**